codetransformer/transformers/exc_patterns.py

```diff
@@ -0,0 +1,51 @@
+from sys import exc_info
+
+from dis import dis
+
+from codetransformer.core import (
+    CodeTransformer,
+    Instruction,
+    ops,
+)
+
+
+def match(match_expr, exc_info):
+    """
+    Called to determine whether or not an except block should be matched.
+
+    True -> enter except block
+    False -> don't enter except block
+    """
+    return (
+        type(match_expr) == exc_info[0] and
+        match_expr.args == exc_info[1].args
+    )
+
+
+class PatternMatchedExceptions(CodeTransformer):
+
+    def visit_COMPARE_OP(self, instr):
+        if instr.arg == 10:
+            yield Instruction(ops.ROT_TWO).steal(instr)
```

Comment:
Do you want to review the instruction branch so we can use the new instruction stuff